graylog2-web-interface/src/@types/graylog-web-plugin/index.d.ts

```diff
@@ -73,7 +73,11 @@ interface PluginCloud {
     onCreate: (formData: { [name: string ]: string }) => { [name: string]: string };
   };
 }
-
+interface InputConfiguration {
+  type: string;
+  component: React.ComponentType<{}> | null;
+  embeddedComponent: React.ComponentType<{}> | null;
```

Comment:
Existing third party custom Input UIs may not set an `embeddedComponent`, so this should be optional:
```suggestion
  embeddedComponent?: React.ComponentType<{}>;
```